src/Angular.js

```diff
@@ -1031,6 +1031,24 @@ function copy(source, destination, maxDepth) {
 
       case '[object Blob]':
         return new source.constructor([source], {type: source.type});
+      case '[object Map]':
+        // If we're in this case we know the environment supports Map
+        /* eslint-disable no-undef */
+        var copiedMap = new Map();
+        /* eslint-enable */
+        source.forEach(function(value, key) {
```

Comment:
If I understand this correctly, IE11 does not support Map and Set constructor with arguments ? https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Map